crates/pathfinder/src/state/sync.rs

```diff
@@ -200,6 +201,18 @@ where
         Ok(l2_head)
     })?;
 
+    let gateway_latest = sequencer
+        .head()
+        .await
+        .context("Fetching latest block from gateway")?;
+
+    let (tx_latest, rx_latest) = tokio::sync::watch::channel(gateway_latest);
+    let mut latest_handle = tokio::spawn(l2::poll_latest(
+        sequencer.clone(),
+        head_poll_interval,
```

Comment:
In the original L2 producer, this was being polled at 1s. I'm wondering if we shouldn't follow suit here? Currently this defaults to 5s in the config, which is probably too slow?

Reply:
Yeah, that's a bit too low I think. Can't we just change that default to something more reasonable? (Like: ~2s)